modules/bash-commons/src/array.sh

```diff
@@ -16,6 +16,28 @@ function array_contains {
   return 1
 }
 
+# Splits the given $string into an array of elements based on the given $separator
+#
+# Examples:
+#
+# array_split "a,b,c", ","
+#   Returns: ("a" "b" "c")
+#
+# Hint:
+# When calling this function, use the following construction: ary=( $(array_split "a,b,c", ",") )
+#
+# Sources:
+# - https://stackoverflow.com/a/15988793/2308858
+function array_split {
+  local -r separator="$1"
+  local -r str="$2"
+  local ary=()
```

Comment:
`local -a` for arrays. Works around an odd 18.04 bug that breaks `local` with arrays without that flag.